### DISCUSSION TITLE
Hints and conditions and a question

### DISCUSSION THREAD

**Discussion by SebastianStehle** [hakimio/ngx-ui-tour#134] [General]
Hi,

Nice library. First I had a look to intro.js (https://introjs.com/), but this library looks promising as well.

I am looking for some features and would like to know if you have planned them. These ideas are basically from this demo: https://userflow.com/demo

1. Support for hints? Is this planned or would you accept contributions? See screenshot 1
2. Support for conditions? e.g. when we wait for a user input? I guess I can already implement that today, but I would prefer the declarative approach that you use right now.

I also have a question:

Can I use the library without any of the supported libraries? I guess I just have to implement a template and that's it?

![image](https://github.com/hakimio/ngx-ui-tour/assets/1236435/1b9c8a8a-b170-4561-9c79-26501ef1725c)

**Comment by hakimio** [comment]
Hi,

No, this library doesn't have support for hints or conditions and I don't plan to implement either. If you want hints, you can try [ngx-guided-tour](https://github.com/lsqlabs/ngx-guided-tour). If you need conditions, you can use [bdc-walkthrough](https://github.com/Broadcom/bdc-walkthrough).
If you can implement any of those features in a simple, elegant way, I might consider including them, but no promises.

You can use the core library (`ngx-ui-tour-core`) if you want to implement your own UI and don't like any of the existing options. Check other implementations for reference.
